MDHelpers/MDStatics.cs

```diff
@@ -462,14 +522,23 @@ public static MethodInfo GetMethodInfo(Node Node, string Method, params object[]
                         List<Type> CandidateSignature = new List<Type>();
                         CandidateParams.ToList().ForEach(param => CandidateSignature.Add(param.ParameterType));
 
+                        MDLog.Debug(LOG_CAT, $"Evaluating {CandidateMethod.Name} ({GetParametersAsString(CandidateSignature.ToArray())})");
+
                         bool IsCompatible = true;
 
                         for (int i = 0; i < Signature.Count; ++i)
                         {
                             Type SignatureType = Signature[i];
                             Type CandidateType = CandidateSignature[i];
-                            if (SignatureType.IsCastableTo(CandidateType) == false)
+                            bool isNullable = CandidateType.IsNullable();
+                            if (CandidateType.IsNullable() && (Parameters == null || Parameters[i] == null))
                             {
+                                // If the parameter is null and type is nullable that is fine
+                                continue;
```

Comment:
Instead of having a separate block I think it's more readable to combine the conditions:

```cs
if ((CandidateType.IsNullable() && (Parameters == null || Parameters[i] == null)) == false
    && (SignatureType.IsCastableTo(CandidateType) == false))
```